docs/3-implementing.md

```diff
@@ -66,3 +73,55 @@ plain field (`foo`) will be of type `keyword`, with no nested field.
 Despite the fact that IDs are often integers in various systems, this is not
 always the case. Since we want to make it possible to map as many data sources
 to ECS as possible, we default to using the `keyword` type for IDs.
+
+## Using non-ECS fields in your event stream
+
+The goal of ECS is to define a common set of fields to make working across various
+streams easier. Each stream will have their own particularities, however, and ECS
```

Comment:
What do you mean by streams here?

Reply:
Event streams. An abstract\* concept of a whole bunch of events of a given kind. 

\* I say abstract because it's not necessarily tied to an ElasticSearch index. For example, the way beats does indexes, all event streams (e.g. Apache logs, system logs, etc.) picked up by Filebeat would be in the same index.